php/tests/GeneratedClassTest.php

```diff
@@ -27,6 +27,17 @@ public function __construct($data = null) {
     }
 }
 
+# This is not allowed, but we at least shouldn't crash.
+class TestMessageMockProxy extends TestMessage {
```

Comment:
nit: the other classes in this class place their opening parenthesis on a new line